index.html

```diff
@@ -0,0 +1,138 @@
+                    positive change with people from over 80 countries is taking place in February, in Australia.
+                </p>
+                <h3>
+                    2023.02.29-31
+                </h3>
+                <p id="address">
+                    @ Melboure Information University, Mcdonalds Building
+                </p>
+            </div>
+
+        </section>
+
+
+        <section class="about" id="about">
+            <h1>Main Programs</h1>
+            <hr />
+                    <div id="programs" aria-label="Lecture">
+                        <div class="program">
+                            <img src="./img/lecture.png" alt="lecture Logo" />
+                            <h2>Lecture</h2>
+                            <p>Listen to the speakers from various countries
+                                about the messages of sharing and opening.
+                            </p>
+                        </div>
+                        <div class="program">
+                            <img src="./img/route.png" alt="route Logo" />
+                            <h2>Route</h2>
+                            <p>Easy to find in downtown Melboure with
+                                lots of attractions to visit during your 
+                                spare time.
+                            </p>
+                        </div>
+                        <div class="program">
+                            <img src="./img/hotel.png" alt="hotel Logo" />
+                            <h2>Hospitality</h2>
+                            <p>Lots of choices of outstanding hotels 
+                                 to satisfy your needs during Conference.
+                            </p>
+                        </div>
+                        <div class="program">
+                            <img src="./img/workshop.png" alt="workshop Logo" />
+                            <h2>Workshop</h2>
+                            <p>Over 50 different workshops can be found in-person and 
+                                online to learn from different field of science research 
+                                at the cutting edge this year.
+                            </p>
+                        </div>
+                        <div class="program">
+                            <img src="./img/youtube.png" alt="youtube Logo" />
+                            <h2>Lecture</h2>
+                            <p>Cannot come in-person? No worries! We provide stream live
+                                on Youtube and you can interact with us during our scheduled
+                                Conference time.
+                            </p>
+                        </div>
+                    </div>
+        </section>
+        <section class="works" id="works">
+            <div class="works-desktop">
+                <h1 aria-label="My Recent works">Featured Speakers</h1>
```

Comment:
- [ ] The featured speakers too need to be aligned as it is on the Behance design.
- [ ] Kindly follow the font of the Behance as the image in this section appears bigger than how it is on behance.

## Your design
![image](https://user-images.githubusercontent.com/86200356/232959248-16826ac3-df2e-4be2-bc9c-e63c7c233c5a.png)

## Behance Design
![image](https://user-images.githubusercontent.com/86200356/232959420-db02bedf-456d-41bc-b5fa-4b7c34f9485f.png)